src/app/shell/provider/provider-config/provider-config.component.scss

```diff
@@ -1,21 +1,387 @@
-form{
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+input::-webkit-outer-spin-button,
```

Comment:
what exactly are you trying to achieve by using -webkit-outer-spin-button pseudo class?
this class is non-standard and not supported by almost all browsers